pkg/csi/service/common/vsphereutil.go

```diff
@@ -373,7 +373,7 @@ func DetachVolumeUtil(ctx context.Context, manager *Manager,
 func DeleteVolumeUtil(ctx context.Context, volManager cnsvolume.Manager, volumeID string, deleteDisk bool) error {
 	log := logger.GetLogger(ctx)
 	var err error
-	log.Debugf("vSphere Cloud Provider deleting volume: %s with deleteDisk flag: %t", volumeID, deleteDisk)
+	log.Debugf("vSphere CNS driver is deleting volume: %s with deleteDisk flag: %t", volumeID, deleteDisk)
```

Comment:
It is ok to remove `vSphere CNS driver is` from the log message.

Reply:
There are multiple places with  prefix: `vSphere CSI driver is`. I think it's ok to have this.
Let me know if you still think we should remove

Reply:
then it is fine.